.github/workflows/npmjs.yml

```diff
@@ -15,19 +15,14 @@ jobs:
                 node: ['14', '16']
         steps:
             - uses: actions/checkout@v2
-            - uses: actions/cache@v2
-              id: cache
-              with:
-                  path: node_modules
-                  key: ${{ matrix.node }}-${{ hashFiles('**/package-lock.json') }}
```

Comment:
When this was added, it helped to speed up most CI runs by not rebuilding `libxslt` and `node1-libxmljs-myh`. After adding `playwright`, I saw errors in CI runs with cache hits, because the browsers are installed outside of `node_modules`. It certainly could have been addressed with additional caching, but I felt the install time is fast enough now that the benefits of this cache were no longer worth the effort.